src/index.html

```diff
@@ -7,12 +7,60 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,300;1,500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="product image"
+        class="card__img"
+      />
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__review-amount">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-title">Price:</p>
+        <p class="card__price-amount">$2,199</p>
+      </div>
+
+      <a
+        href="#"
+        class="card__buy"
+        data-qa="hover"
+      >
```

Comment:
The 'data-qa' attribute with the value 'hover' seems to be misleading. If it's meant for testing purposes, ensure that the value corresponds to the element's purpose or action.

Reply:
One of requirement to the task:
- add data-qa="hover" attribute to the link BUY